node/cmd/txverifier/sui.go

```diff
@@ -164,7 +164,7 @@ func runTransferVerifierSui(cmd *cobra.Command, args []string) {
 			logger.Info("Context cancelled")
 		case <-ticker.C:
 			// Fetch new events
-			resp, err := suiApiConnection.QueryEvents(eventFilter, "null", 25, true)
+			resp, err := suiApiConnection.QueryEvents(ctx, eventFilter, "null", InitialEventFetchLimit, true)
```

Comment:
`"null"` is used everywhere for this parameter -- is this correct or should it update under some conditions?

If it's always `"null"` then maybe the QueryEvents function should remove it as an argument.